templates/test/dev/cluster-template-custom-builds-windows.yaml

```diff
@@ -3,13 +3,15 @@ kind: Cluster
 metadata:
   labels:
     cni: ${CLUSTER_NAME}-flannel
+    csi-proxy: enabled
```

Comment:
this file can be deleted, it was only used by dockershim e2e tests

Reply:
done